.buildkite/pipeline_fullchip.yml

```diff
@@ -21,6 +21,9 @@ env:
   # Can use this to change target mflowgen branch
   # OVERRIDE_MFLOWGEN_BRANCH: glob-prob
 
+  # Newer dockers use lake "sparse_strawman" branch offshoots :(
+  RTL_DOCKER_IMAGE: 'stanfordaha/garnet@sha256:dd688c7b98b034dadea9f7177781c97a7a030d737ae4751a78dbb97ae8b72af4'
```

Comment:
Note this change applies only to buildkite "pipeline_fullchip" builds, which only run on TSMC branches